src/service/get-featured-for-content.test.ts

```diff
@@ -3,6 +3,11 @@ import { Featured, FeaturedItem } from '../interfaces/featured';
 import { getFeaturedForContent } from './get-featured-for-content';
 import * as getFeaturedForTopicModule from './get-featured-for-topic';
 
+jest.mock('./get-featured-for-topic', () => ({
```

Comment:
Seems like upgrading SWC (a side effect of upgrading next) introduces strange behavior with `jest.spyOn`. [This project](https://github.com/okp4/ui/pull/457) seems to have had a similar issue after upgrading SWC to `1.2.242`. This work-around of mocking the module exports with their actual values was suggested [here](https://github.com/aelbore/esbuild-jest/issues/26#issuecomment-968853688) and it seems to do the trick.